pkg/models/line_of_accounting_test.go

```diff
@@ -0,0 +1,22 @@
+package models_test
+
+import (
+	"github.com/transcom/mymove/pkg/factory"
+)
+
+func (suite *ModelSuite) Test_AllFieldsOptionalCanSave() {
```

Comment:
Add test for querying the saved models

```suggestion
func (suite *ModelSuite) Test_AllFieldsOptionalCanSaveAndQuery() {
```

Reply:
https://github.com/transcom/mymove/pull/11090/commits/b0a124db8b8cd052966cbeeb0fcf6a16019b17cc